requre/cassette.py

```diff
@@ -99,6 +99,19 @@ def obj_cls(self, value):
         self._obj_cls = value
 
 
+def _get_module_name(obj: Any) -> Optional[str]:
+    module = inspect.getmodule(obj)
+    if module:
+        return module.__name__
+    elif (
+        hasattr(obj, "__class__")
+        and obj.__class__.__name__ == "builtin_function_or_method"
+    ):
+        return obj.__class__.__name__
+    else:
+        return ""
```

Comment:
When does this happen?

Reply:
@lachmanfrantisek I do not know, but better to have safe option than let it fail somehow I think.

Reply:
Ok, maybe something implemented in C without proper Python binding.